src/filesys.cpp

```diff
@@ -860,17 +861,17 @@ bool safeWriteToFile(const std::string &path, std::string_view content)
 	HANDLE tmp_handle = CreateFile(
 		tmp_file.c_str(), GENERIC_WRITE, 0, nullptr, CREATE_ALWAYS, FILE_ATTRIBUTE_NORMAL, nullptr);
 	if (tmp_handle == INVALID_HANDLE_VALUE) {
+		errorstream << "Failed to open file: " << porting::ConvertError(GetLastError()) << std::endl;
 		return false;
 	}
 	DWORD bytes_written;
 	tmp_success = (WriteFile(tmp_handle, content.data(), content.size(), &bytes_written, nullptr) &&
```

Comment:
I think we should log errors for these two functions too. They could cause the error in the linked issue.